src/test/kotlin/io/github/mojira/arisa/modules/FutureVersionModuleTest.kt

```diff
@@ -276,6 +276,22 @@ class FutureVersionModuleTest : StringSpec({
         result.shouldBeRight(ModuleResponse)
     }
 
+    "should not resolve if there is another version" {
+        val module = FutureVersionModule("message")
+        val issue = mockIssue(
+            created = FIVE_SECONDS_AGO,
+            affectedVersions = listOf(FUTURE_VERSION, RELEASED_VERSION),
```

Comment:
This doesnt really test the behavour(as there is no easy way after the caching work), but it serves as a spec of what themodule does

Reply:
You should be able to set the resolveAsAwaitingResponse function and assert that it is not being called, like we usually do.

Reply:
Already done XD